clang/include/clang/Basic/AArch64SVEACLETypes.def

```diff
@@ -57,6 +57,10 @@
 //  - IsBF true for vector of brain float elements.
 //===----------------------------------------------------------------------===//
 
+#ifndef SVE_TYPE
```

Comment:
I thought of adding this in one of the previous refactorings, in the end decided to omit it as too error prone - the compiler
is going to happily compile everything that includes this file even if we forget to define some macro.
It still not ideal, because one can define `SVE_TYPE` at the place of the include and *still* forget  to define
some of the other macros, but at least requires *some* thought at each individual usage.